roles/vgpu/vars/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+
+vgpu_mig_enabled: "{{ vgpu_definitions | selectattr('mig_definitions', 'defined') | length > 0 }}"
```

Comment:
Shouldn't mig_definitions be replaced with mig_devices ?

Reply:
Think I got this one :thinking: